python/tk_multi_workfiles/actions/open_file_action.py

```diff
@@ -240,13 +240,18 @@ def _open_in_current_work_area(self, src_path, src_template, file, src_work_area
 class ContinueFromFileAction(OpenFileAction):
     """
     """
-    def __init__(self, label, file, file_versions, environment):
+    def __init__(self, label, file, file_versions, environment, next_version=None):
         """
         """
-        # Q. should the next version include the current version?
-        all_versions = [v for v, f in file_versions.iteritems()] + [file.version]
-        max_version = max(all_versions)
-        self._version = max_version+1
+
+        if next_version is not None:
+            self._version = next_version
+        else:
+            # Q. should the next version include the current version?
+            all_versions = [v for v, f in file_versions.iteritems()] + [file.version]
```

Comment:
can you turn `[v for v, f in file_versions.iteritems()]` into just `file_versions.keys()`?